src/generics/utils.js

```
@@ -893,6 +895,72 @@ function buildUrl(baseUrl, endpoint, queryParams = {}, idParam = null) {
+}
+
+/**
+ *  Downloads a file from a URL and saves it to a local file
+ * @function
+ * @name downloadFile
+ * @param {String} url - Download url
+ * @param {String} filePath - Local storage path
+ * @returns {Promise<String>}
+ */
+
+async function downloadFile(url, filePath) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'downloadFile'. Consider to delete it or rename it.

'downloadFile' is defined here:


(lint/suspicious/noRedeclare)